### ISSUE TITLE
[Suggestion]: Add an item blacklist

### ISSUE THREAD

**Issue by SattesKrokodil** [Draylar/inmis#116]
would be cool to have a config where you can blacklist certain items from being put into backpacks (=

**Comment by ouroborus** [comment]
Yes, please. This would allow blacklisting of items from other backpack mods.

**Comment by Ghezra** [comment]
Just had a crash on a multiplayer server because a player put many books inside a travellers backpack, and put many of those backpacks into an inmis backpack, and then had many inmis backpacks in a travellers backpack, and loops until the chunk was corrupted because too much metadata.

Definitely would like a way to blacklist items.
